README.md

```diff
@@ -16,9 +16,23 @@ In alphabetical order:
 $ pip install visumorph
 ```
 
-## Usage
+## Features
 
-- TODO
+VisuMorph includes a variety of functions for image manipulation:
+
+- **Image Class(`Image`)**: A core class that represents images, facilitating the application of transformations and adjustments.
```

Comment:
I think we better not include this as this is a class (not a function) and it is not "user-facing".

```suggestion
```